perf/benchmark/configs/istio/telemetryv2_stats/cpu_mem.yaml

```diff
@@ -1,5 +1,5 @@
-# Configuration Set5: CPU and memory with telemetryv2 using NullVM.
-telemetry_mode: "v2-nullvm"
+# Configuration Set5: CPU and memory with telemetryv2 using NullVM with stats envoyfilter.
+telemetry_mode: "v2-stats-nullvm"
```

Comment:
only has stats filter, I though this have both stats and metadata exchange filters?

Reply:
metadata-exchange is required for both cases by now, we may just need to distinguish between prometheus and stackdriver

Reply:
v2 stats should have mx too. It is important to benchmark the actual configuration that users use.